tools/config/stix-custom.yml

```diff
@@ -0,0 +1,132 @@
+title: Additional STIX mapping for future use
+backends:
+  - stix
+order: 10
+fieldmappings:
+  record_type:
+    - x-dns:record_type
+  requestParameters.attribute:
+    - x-cloud:request_parameters
+  responseElements.publiclyAccessible:
+    - x-cloud:publicly_accessible
+  errorMessage:
+    - x-error:message
+  errorCode:
+    - x-error:code
+  responseElements:
+    - x-cloud:response_elements
+  requestParameters.userData:
+    - x-cloud:request_parameters
+  AccessMask:
+    - x-windows:accessmask
+  Accesses:
+    - x-windows:accesses
+  CallTrace:
+    - x-windows:calltrace
+  DestinationIsIpv6:
+    - x-windows:destisipv6
+  ErrorCode:
+    - x-error:code
+  ExtendedErrorCode:
+    - x-error:code
+    - x-error:id
+  GrantedAccess:
+    - x-windows:grantedaccess
+  GroupDomain:
+    - x-group:domain
+  GroupID:
+    - x-group:id
+  GroupName:
+    - x-group:name
+  GroupSecurityID:
+    - x-group:security_id
+  IMPHash:
+    - x-windows:imphash
+  Imphash:
+    - x-windows:imphash
+  ImageTempPath:
+    - process:binary_ref.x_temp_path
+  InitiatedConnection:
+    - x-windows:initiatedconnection
+  Initiated:
+    - x-windows:initiatedconnection
+  IntegrityLevel:
+    - x-windows:integritylevel
+  LogonType:
+    - x-windows:logontype
+  ObjectName:
+    - x-windows:objectname
+  ObjectType:
+    - x-windows:objecttype
+  PipeName:
+    - x-windows:pipename
+  QueryName:
+    - x-windows:queryname
+  QueryResults:
+    - x-windows:queryresults
+  QueryStatus:
+    - x-windows:querystatus
+  ShareName:
+    - x-windows:sharename
+  SharePath:
+    - x-windows:sharepath
+  Signature:
+    - x-windows:signature
+  SignatureStatus:
+    - x-windows:signaturestatus
+  Signed:
+    - x-windows:signed
+  SourceImage:
```

Comment:
possible use of `process:binary_ref.name`

Reply:
Done